Gemfile.lock

```diff
@@ -0,0 +1,179 @@
+GIT
```

Comment:
It is not recommended when we are building gem. 
Check here http://stackoverflow.com/a/4151540/1120530